sktime/base/_panel/knn.py

```diff
@@ -0,0 +1,103 @@
+"""KNeighbors Time Series Estimator base class."""
+
+__author__ = ["fkiraly", "Z-Fran"]
+__all__ = ["BaseKnnTimeSeriesEstimator"]
+
+
+class BaseKnnTimeSeriesEstimator:
```

Comment:
make this a private class, start it with underscore, or it might be picked up by the tests.